converters/toZigbee.js

```diff
@@ -2364,6 +2364,84 @@ const converters = {
             await entity.read('haElectricalMeasurement', [0xf000, 0xf001, 0xf002]);
         },
     },
+    tuya_led_control: {
```

Comment:
For all the added converters, also return the new state like: https://github.com/Koenkk/zigbee-herdsman-converters/blob/ba3505b0c841caa54f05efa97479ae031e343bb6/converters/toZigbee.js#L636